src/core/comm/coll.cc

```diff
@@ -245,20 +247,28 @@ int collInit(int argc, char* argv[])
   int provided, init_flag = 0;
   CHECK_MPI(MPI_Initialized(&init_flag));
   if (!init_flag) {
-    log_coll.fatal(
-      "MPI has not been initialized, it should be initialized by "
-      "the networking backend");
-    LEGATE_ABORT;
-  } else {
-    int mpi_thread_model;
-    MPI_Query_thread(&mpi_thread_model);
-    if (mpi_thread_model != MPI_THREAD_MULTIPLE) {
+    char* network = getenv("LEGATE_NEED_NETWORK");
```

Comment:
`LEGATE_NEED_NETWORK` will be either undefined, 0 or 1, it's not the number of nodes.